src/Compilers/CSharp/Test/Semantic/Semantics/DeconstructionTests.cs

```diff
@@ -1327,28 +1325,16 @@ static void Main()
             comp.VerifyDiagnostics(
                 // (6,13): error CS8059: Feature 'tuples' is not available in C# 6.  Please use language version 7 or greater.
                 //         var (x1, x2) = Pair.Create(1, 2);
-                Diagnostic(ErrorCode.ERR_FeatureNotAvailableInVersion6, "(x1, x2)").WithArguments("tuples", "7").WithLocation(6, 13),
```

Comment:
I think there is a bug with the diffing logic when the test fails. The locations are only printed out correctly when no diagnostics are expected.